samples/bootloader/README.rst

```diff
@@ -218,12 +218,18 @@ Firmware versions using the |NSIB| are kept in the form of a *monotonic counter*
 Counter values are kept as slots in the flash memory, with each new counter value occupying a new slot.
 See :kconfig:option:`CONFIG_SB_MONOTONIC_COUNTER` for more details.
 
-To set the counter value of an application, build it using the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option:
+You can set the counter value for different elements in the bootloader sequence.
+Using :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option in ``prj.conf`` will allow you to change the version of the application image.
+However, in order to specify :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` for the MCUboot image you have to create a ``child_image`` folder in the project directory.
```

Comment:
Strictly speaking you don't _have_ to create a `child_image` folder, but _I_ think that is the most relevant method for this specific use-case (monotonic counter version for mcuboot).

```suggestion
However, in order to specify :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` for the MCUboot image:
* Create a ``child_image`` folder in the project directory if it doesn't exist already.
* Create a file named ``mcuboot.conf`` in the ``child_image`` directory.
* Add the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option to this file.
```

Reply:
updated